### PR TITLE
avro file type support added

### PR DESCRIPTION
**Pull request by IgorWiecz**
Avro file type support added, steps in contributing.md file followed.

### CODE REVIEW

File: core.d.ts [hunk 1]
```diff
@@ -167,6 +168,7 @@ export type MimeType =
 	| 'image/vnd.ms-photo'
 	| 'image/vnd.adobe.photoshop'
 	| 'application/x-indesign'
+	| 'application/avro'
```
Comment (sindresorhus) [inline core.d.ts:171]: Should be added to the end.

File: core.d.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import type {Readable as ReadableStream} from 'node:stream';
 import type {ITokenizer} from 'strtok3';
 
 export type FileExtension =
+	| 'avro'
```
Comment (sindresorhus) [inline core.d.ts:5]: Should be added to the end.

Comment (IgorWiecz) [reply core.d.ts]: Done.

File: core.d.ts (hunk 1 above)
Comment (IgorWiecz) [reply core.d.ts]: Done.

File: core.js [hunk 3]
```diff
@@ -292,6 +299,13 @@ class FileTypeParser {
 			};
 		}
 
+		if (this.checkString('avro')) {
```
Comment (sindresorhus) [inline core.js:302]: Can you link to a source for this?

File: supported.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 export const extensions = [
+	'avro',
```
Comment (sindresorhus) [inline supported.js:2]: It was implied that https://github.com/sindresorhus/file-type/pull/597#discussion_r1179941635 applied here too. And the below.

File: core.js (hunk 3 above)
Comment (sindresorhus) [reply core.js:302]: I don't see it written anywhere there that the file header must start with `avro`.

Comment (IgorWiecz) [reply core.js]: I've also noticed people use .avsc and .acme, but I don't quite understand the difference

**Pull request merged**